src/ldc/eh/msvc.d

```diff
@@ -109,12 +111,16 @@ extern(C) void _d_throw_exception(Object e)
         if (!old_terminate_handler)
             old_terminate_handler = set_terminate(&msvc_eh_terminate);
     }
-    auto t = cast(Throwable) e;
-    exceptionStack.push(t);
+    
+    auto throwable = cast(Throwable) e;
+    exceptionStack.push(throwable);
+
+    if (throwable.info is null && cast(byte*)throwable !is typeid(throwable).init.ptr)
+        throwable.info = _d_traceContext();

```

Comment:
--DRT options can also be embedded in the executable or picked up from the environment. See https://dlang.org/spec/garbage.html#gc_config